pages/index.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+  <div class="container">
+    <h1 class="title">Pokemon Familiy</h1>
+    <router-link to="/pokemons/favourites" class="favourites-link"
+      >Go to Favourites</router-link
+    >
+    <template v-if="data">
+      <div
+        v-for="page in data.pages"
+        :key="page.page"
+        class="pokemon-list-container"
+      >
+        <div
+          v-for="pokemon in page.pokemons"
+          :key="pokemon.id"
+          class="pokemon-item"
+          @click="goToDetail(pokemon.id)"
```

Comment:
Why use a click on a div & not use link? 
A click on a div is not semantic html